docs/model-dev-guide/best-practices/_index.rst

```diff
@@ -1,75 +0,0 @@
-################
- Best Practices
-################
-
-***************************************
- General Tips for the Trial Definition
-***************************************
-
-Do:
-
--  Use framework abstractions to implement learning rate scheduling instead of directly changing the
-   learning rate. See `tf.keras.optimizers.schedules.LearningRateSchedule
-   <https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/LearningRateSchedule>`__
-   and :class:`determined.pytorch.LRScheduler` as examples.
-
--  For code that needs to download artifacts (e.g., data, configurations, pretrained weights),
-   download to a `tempfile.TemporaryDirectory <https://docs.python.org/3/library/tempfile.html>`__
-   unique to the Python process. This will avoid race conditions when using distributed training, in
-   which Determined executes multiple Python processes in the same task container.
-
-.. include:: ../../_shared/note-dtrain-learn-more.txt
-
-Do not use instance attributes on a trial class to save any state over time (e.g., storing metric
```

Comment:
this is true but this isn't enough to justify keeping the whole document for just this